cart/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django.conf import settings
+from products.models import Product
+
+
+class Cart(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```

Comment:
What is the idea of having this model? Why can't the CartItem have direct FK to the User model?

Reply:
approved eventually